VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/wr-participant-list-shared.component.ts

```diff
@@ -26,9 +26,11 @@ export abstract class WRParticipantStatusListDirective implements DoCheck {
 
     nonJudgeParticipants: ParticipantResponse[];
     judge: ParticipantResponse;
+    staffMember: ParticipantResponse;
     endpoints: VideoEndpointResponse[];
     observers: ParticipantResponse[];
     panelMembers: ParticipantResponse[];
+    staffMembers: ParticipantResponse[];
```

Comment:
Any reason we have a list of staffMembers? My understanding was that it would be just 1

Reply:
yeah I know, I can use another way to handle this